packages/near-fast-auth-signer/src/utils/multi-chain/relayer/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface FunctionCall {
+  FunctionCall: {
+    method_name: string;
+    args: string;
+    gas: number;
+    deposit: string;
+  };
+}
+
+export type Actions = FunctionCall;
+
+export interface DelegateActionRelayerFormat {
```

Comment:
Same for `import { DelegateAction } from '@near-js/transactions';`

Reply:
Same reason as the previous comment, the types are different

`export declare class DelegateAction extends Assignable {
    senderId: string;
    receiverId: string;
    actions: Array<Action>;
    nonce: BN;
    maxBlockHeight: BN;
    publicKey: PublicKey;
}`

Reply:
same as above